orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -89,50 +89,50 @@ def state_dict(self, destination=None, prefix='', keep_vars=False):
         """Override original method to delegate execution to the base module"""
 
         # Override the state_dict() method so that the state dict key names
-        # do not contain the _flattened_module._base_module prefix
-        return self._flattened_module._base_module.state_dict(
+        # do not contain the _flattened_module._original_module prefix
+        return self._flattened_module._original_module.state_dict(
```

Comment:
Out of curiosity, is there a reason to prefer returning `self._flattened_module._original_module....` and not `self._original_module....`?

Reply:
Originally introduced by https://github.com/microsoft/onnxruntime/pull/7220. Using `self._original_module` still passes `test_state_dict`, so I am making the change here (despite the unrelated